Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Pull and use the official Docker Hub Ubuntu 14.04 base image
+FROM ubuntu:14.04
```

Comment:
Suggest ubuntu:latest or as mentioned in comments a minimal flash-nginx. This looks promising: https://hub.docker.com/r/tiangolo/uwsgi-nginx-flask/